package_manifest.yaml

```diff
@@ -32,6 +32,29 @@ all:
   children:
     packages: {}
     repoclosures: {}
+    copr_projects: {}
+
+copr_projects:
+  vars:
+    copr_project_user: 'theforeman'
```

Comment:
this is now set here and on the `foreman_client_packages` entry -- should it be set somewhere more globally? I don't expect it to be set to different values for different targets anyway?